Integrations/integration-nmap.yml

```diff
@@ -0,0 +1,98 @@
+description: Run nmap scans with the given parameters
+configuration: []
+script:
+  script: |
+    from libnmap.process import NmapProcess
+    from libnmap.parser import NmapParser
+    from libnmap.reportjson import ReportEncoder
+
+    if demisto.command() == 'test-module':
+        demisto.results('ok')
+        sys.exit(0)
+    if demisto.command() == 'nmap-scan':
+        nm = NmapProcess(argToList(demisto.args()['targets']), options=demisto.args()['options'])
+        rc = nm.run()
+        if rc != 0:
+            demisto.results({'Type': entryTypes['error'], 'ContentsFormat': formats['text'], 'Contents': 'Unable to execute - ' + nm.stderr})
+            sys.exit(0)
+        r = NmapParser.parse(nm.stdout)
+        md = '## ' + r.summary + '\n\n'
+        hosts = []
+        for host in r.hosts:
+            h = {}
+            if len(host.hostnames):
+                tmp_host = host.hostnames.pop()
+                h['Hostname'] = tmp_host
+            else:
+                tmp_host = host.address
+
+            h['Address'] = host.address
+            h['Status'] = host.status
+            svc = []
+            md += "- Nmap scan report for {0} ({1})".format(tmp_host, host.address) + '\n'
```

Comment:
If tmp_host == host address this line will look wierd

Reply:
This is the nmap style reporting but I see what you mean